scapy/fields.py

```diff
@@ -556,6 +563,9 @@ def getfield(self, pkt, s):
         l = self.length_from(pkt)
         return s[l:], self.m2i(pkt,s[:l])
 
+def _toHex(x):
```

Comment:
rename it to `to_hex()` to conform with other Scapy function name. This could go in utils.py